lab_5_scrapper/scrapper.py

```diff
@@ -177,6 +314,7 @@ def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> No
             article_soup (bs4.BeautifulSoup): BeautifulSoup instance
         """
 
+
```

Comment:
you have written so much code, but where is demonstration of its work?

Reply:
Done